docs/_docs/Instructors/04-Recommendations-High-Quality-materials.md

```diff
@@ -0,0 +1,80 @@
+
+## How much theory? 
+There is a delicate trade-off between making sure the students know why they are using something, and feeding them theory for theory's sake. Be practical, and think whether a piece of theoretical knowledge has helped you significantly to use a particular concept. 
+
+As an example, having an intuition of Gradient Descent helps understand why a Logistic Regression needs to be normalized, and is therefore worth including. On the other hand being able to derive the Cross Entropy error function is mostly fun-for-nerds, and arguably does not particularly enhance our ability to use it in practice. 
+
+Ensure that everything we teach is there to make them be able to use the concepts, while avoiding over-correcting and only teaching "how to do stuff". Never teach something just because you had to suffer through it yourself, or because you happen to find it cool, as remember that you have years of experience, and a completely different base. Whenver in doubt, ask the head of Curriculum (in our case João Ascensão). 
+
+## Add an optional section at the end 
+At the end of your Learning Notebook, make a section entitled _"To learn more (Optional)"_, where you put links to further sources. Two rules here: 
+> 1. Don't put links to anything you haven't read yourself, and really found useful (no link dumping)
+> 2. Use markdown links, and use the description to say why the link is worth reading
+(e.g. [Step by step way to calculate Variance by hand](https://www.wikihow.com/Calculate-Variance))
+
+## Hide peripheral support code 
+When making notebooks, it is inevitable that you will need to do a bit of extra code. This might be a few lines of pre-processing to get or make data to show the new concept (e.g. making a new DataFrame, or using `get_dummies`), or some matplotlib code to produce a good intuitive plot. 
+
+It is essential however that students be isolated from concepts that aren't the ones being taught. In other words, that they don't spend time looking at matplotlib code, because they can't tell what's core from what is peripheral. For this reason, it is best practice to use a small `utils.py` file at the top level where you hide all your code. Don't worry about making this clean or understandable, the student's shouldn't be peaking here. Here is an [example utils file](https://github.com/PedroGFonseca/preping_template_learning_unit/blob/master/SLU6_Intermediate_Statistics/utils.py), in this case for intermediate statistics. 
+
+## Data, images, and other supporting material
+
+If you include a dataset, be sure that
+
+1. It is small (1mb is ideal, definitely not more than 5mb)
+1. It is stored in a `data` directory
+
+If you need to include images or other media and you want to check them into the repo
+
+1. Again, make sure they are small (<1mb, definitely not more than 5mb)
+1. Put them in a `media` filder
```

Comment:
Alternatively, you can add them to imgur and copy the URL. This is something I tend to do, I'm not sure if it's something worth making official 😅

Reply:
I like to package everything in `media` cause you can go through all notebooks offline if you want to, without loosing anything

Reply:
Very fair point! 😛